plugins/org.jkiss.dbeaver.tasks.ui/src/org/jkiss/dbeaver/tasks/ui/wizard/TaskConfigurationWizardDialog.java

```diff
@@ -248,6 +249,20 @@ public void updateButtons() {
             if (finishButton != null && !finishButton.isDisposed()) {
                 getShell().setDefaultButton(finishButton);
             }
+
+            IWizardPage currentPage = getCurrentPage();
+            if (currentPage != null) {
+                Button nextButton = getButton(IDialogConstants.NEXT_ID);
+                boolean isLogPage = currentPage.getClass().getName().contains("Log");
```

Comment:
I would rather create a method in IWizardPageNavigable that indicates it's the last page.